openbas-front/src/utils/hooks/useDataLoader.js

```diff
@@ -18,7 +18,7 @@ const ERROR_30S_DELAY = 30000;
 let sseClient;
 let lastPingDate = new Date().getTime();
 const listeners = new Map();
-const useDataLoader = (loader = () => {}) => {
+const useDataLoader = (loader = () => {}, refetchArg = []) => {
```

Comment:
Can you explain this changes ?
Thank you

Reply:
Let’s return to the exerciseReportContent folder. I need the useDataLoader function to refetch data every time the report changes. To achieve this, I want to pass a refetch argument into the useEffect hook inside the useDataLoader function. 

By implementing this approach, I ensure that only the necessary data for the report is loaded, rather than fetching all available data. Whenever I update my report configuration, the useDataLoader function will refetch only the specific data required for the updated report.

Reply:
Great, thanks for the explanation